qualitas/templates/dashboards/cnx_repos.html

```diff
@@ -93,58 +93,152 @@
   </script>
 </head>
 
-<body data-feedly-mini="yes">
+<body data-feedly-mini="yes"><!DOCTYPE html>
+<head>
+  <title>CNX Repos Dashboard</title>
+  <meta charset="UTF-8">
+  <meta name=viewport content="width=700">
+  <link rel="shortcut icon" href="newspaper.ico" type="image/x-icon">
+  <style>
+  body {
+    font-family: Noto Sans, Arial, sans-serif;
+    font-size: small;
+  }
+
+  table { width: 700px; }
+
+  .noborder { border:0; }
+
+  td {
+   border: 1px solid gray;
+   text-align: left;
+   vertical-align: top;
+   padding: 7px;
+   width: 50%;
+  }
+  </style>
+
+  <script>
+
+  function getLatestCNXTag(repo) {
+    tags_url = 'https://api.github.com/repos/' + repo + '/tags';
+    var xhr = new XMLHttpRequest();
+    xhr.onreadystatechange = function() {
+      if (xhr.readyState == XMLHttpRequest.DONE) {
+        response = xhr.responseText;
+        resp = JSON.parse(response);
+        latest_tag = resp[0].name;
+
+        newlink = document.createElement('a');
```

Comment:
Let's try to reduce the duplication of effort. I can see updating the DOM with the version number being it's own function. 

Example:

```javascript
function appendVersionLink(id, versionNum, url) {
      let link = document.createElement('a');
      link.innerHTML = versionNum;
      link.setAttribute('target', '_blank');
      link.setAttribute('href', url);
      document.getElementById(id).appendChild(link);
    }
```